converter/internal/staticconvert/testdata_windows/integrations.river

```diff
@@ -23,18 +23,8 @@ http {
 }
 
 prometheus.exporter.windows "integrations_windows_exporter" {
-	enabled_collectors = ["cpu", "cs", "logical_disk", "net", "os", "service", "system"]
-
-	dfsr {
-		sources_enabled = ["connection", "folder", "volume"]
-	}
-
 	exchange { }
 
-	mssql {
-		enabled_classes = ["accessmethods", "availreplica", "bufman", "databases", "dbreplica", "genstats", "locks", "memmgr", "sqlstats", "sqlerrors", "transactions", "waitstats"]
```

Comment:
By default all are enabled https://github.com/grafana/windows_exporter/blob/3391dcec6884e64b0f6face500805c9fbc406b31/collector/mssql.go#L411C1-L411C1

which is matching this list: https://github.com/grafana/windows_exporter/blob/3391dcec6884e64b0f6face500805c9fbc406b31/collector/mssql.go#L74